packages/manager/modules/bm-server-components/src/netboot/controller.js

```diff
@@ -22,11 +24,15 @@ export default class BmServerComponentsNetbootCtrl {
   }
 
   $onInit() {
+    this.currentLanguage =
+      this.$translate.use() || this.$translate.fallbackLanguage();
     this.isNutanixNode = this.isNutanixNode ? this.isNutanixNode : false;
     this.HARDDISK = 'harddisk';
     this.RESCUE = 'rescue';
     this.NETWORK = 'network';
 
+    this.defaultRescue = 'rescue-customer';
```

Comment:
**nitpick:** `rescue-customer` can be defined as a constant in the constants file. And since defaultRescue is a constant, 
```suggestion
    this.DEFAULT_RESCUE = 'rescue-customer';
```